### ISSUE TITLE
I created a whitelister helper for purgecss!

### ISSUE THREAD

**Issue by qodesmith** [FullHuman/purgecss-webpack-plugin#9]
Guys!

TL;DR - [purgecss-whitelister](https://github.com/qodesmith/purgecss-whitelister)

Because purgecss is awesome, it only makes sense to build things around the ecosystem :)

So I was rebuilding my site in React and consuming my own 3rd party libraries. Did all the webpacky stuffs and life was good. But the library I was using had its styles stripped. Ok, the whitelist option to the rescue. Since my library only had a few styles, this wasn't that big a deal. But if I had a tooon of styles that I wanted whitelisted, this would've been a pain in the donkey. [purgecss-whitelister](https://github.com/qodesmith/purgecss-whitelister) to the rescue!  Give it a string or array of strings or globby strings and it will generate an array of names based off the selectors used in that CSS file.

I just _had_ to come tell you guys to see what you thought. Anyway, feel free to close this issue since its not an _issue_, but I just had to let _somebody_ know! Love the work you guys are doing. My CSS bundles are now nano-sized thanks to you. 💯

**Comment by Ffloriel** [comment]
This is great. I believe this is also related to  FullHuman/purgecss-webpack-plugin/issues/3.
I looked also at typer.js, the demo [here](http://aaroncordova.xyz/typer) looks awesome.

**Comment by qodesmith** [comment]
@Ffloriel just left a comment on that issue. Forgot I was there before! Glad you like the library. I'm gonna look at purge's source code. I'd love to be able to contribute.

**Comment by Ffloriel** [comment]
Your contribution is welcomed 🙂  . If you feel interested in purgecss and you would like to contribute more, I can add you to the team as a member.
